src/components/Heading/Heading.stories.tsx

```diff
@@ -16,13 +16,36 @@ export const allHeadings = () => [
     <Heading level={6} id="heading6" text={"Heading 6"} />,
 ];
 
+allHeadings.story = {
+    name: "All Headings",
+    parameters: {
+        design: {
+            type: "figma",
+            url:
+                "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=10975%3A16",
```

Comment:
Does this show up somewhere in storybook?

Reply:
Yes, it shows up in the "Design" tab (work introduced by Monica in #143):

<img width="1212" alt="Screen Shot 2020-06-19 at 2 25 40 PM" src="https://user-images.githubusercontent.com/1720093/85168705-c1f20d80-b238-11ea-80c2-eda3c762e6d9.png">